ChangeLog

```diff
@@ -1,3 +1,16 @@
+Tue Jun 30 2019 Szymon Romik <szymon.romik@intel.com>
+
+	* Version 1.0
+
+ 	This is the first pmemkv-java release which is fully compatible with
+	pmmekv 1.0 and guarantees backward compatibility.
+
+	Major changes:
+	- build system refactor (merged with pmemkv-jni and use mvn only)
```

Comment:
1. perhaps move this line below, it's not as important as e.g. API redesign ;)
2. I personally don't like what's in the parentheses, I'd rewrite it like e.g. "use mvn only; sources of pmemkv-jni merged into this project"

Reply:
1.done
2.done/modified/extended